pkg/cli/delete.go

```diff
@@ -0,0 +1,83 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/QMSTR/qmstr/pkg/service"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+)
+
+var deleteCmd = &cobra.Command{
+	Use:   "delete [type_of_node:attribute:value]",
+	Short: "Delete node from the database",
+	Long:  `Delete the provided node from the database.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		setUpBuildService()
+		setUpControlService()
+		if err := deleteNode(args); err != nil {
+			Log.Fatalf("Delete failed: %v", err)
+		}
+		tearDownServer()
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(deleteCmd)
+}
+
+func deleteNode(args []string) error {
+	if len(args) < 1 {
+		return fmt.Errorf("Please provide the node to be deleted: [type_of_node:attribute:value]")
```

Comment:
There is nothing to format here so errors.New instead of fmt.Errorf should be used.
BUT actually you should remove this check as it is part of ParseNodeID that will throw a ErrEmptyNodeIdent in this case.